hrjobcontract/js/src/job-contract/services/contract.js

```diff
@@ -25,16 +28,18 @@ define([
        * @param {Object} data
        *   The data object as returned by the API
        */
-      function adjustAddPublicHolidaysValue (data) {
-        angular.forEach(data.leave, function (value) {
-          value.add_public_holidays = !!parseInt(value.add_public_holidays);
+      function adjustAddPublicHolidaysValue (contracts, absenceTypes) {
```

Comment:
Please update the doc with new params.

Reply:
Doc is still not correct, it should have `contracts` and `absenceTypes`, not `data`.

Reply:
Apologies.